packages/formation/sass/site/_m-vet-nav.scss

```diff
@@ -120,7 +120,7 @@ body.va-pos-fixed {
   }
 
   .usa-button-secondary {
-    padding: calc(scale-rem(1rem) + 2px);
+    padding: scale-rule(calc(1rem + 2px));
```

Comment:
@it-harrison  At the top of this file, there's a note that says:

```
// appears to be unused by header in vets-website
```

But it seems like it is being used at least for this class? If so, I'm going to move this stylesheet over to the CSS Library.